### DISCUSSION TITLE
New İdea Pls

### DISCUSSION THREAD

**Discussion by iMertKc** [stoatchat/discussions#872] [Feature Suggestions]
Hello, we tested your revolt chat application, we have a big team on discord and since discord is Closed in Turkey, we are thinking of switching to your application, but we saw that there were many deficiencies in terms of usability, we are a team that includes publishers and if there is communication between us and you make developments, we can reach a large audience in Turkey, if you want, we can send you our suggestions after your answer, we wish you a good day.

**Comment by IAmTomahawkx** [comment]
You are welcome to post your requests, but please search the discussions to see if they've already been requested (as most of them probably have).
